library/core/class.format.php

```diff
@@ -1713,6 +1714,49 @@ public static function mentions($Mixed) {
         }
     }
 
+    /**
+     * Optionally replace any embedded images from untrusted sites with links to the image.
+     *
+     * @param string $html String from which you might want to strip out untrusted images.
+     * @return string The string with images replaced by links.
+     */
+    public static function sanitizeImages($html) {
+        // If the config does not have a SanitizeImages entry or it is set to false, pass the HTML to the renderer as is.
+        if (!c('Garden.Format.SanitizeImages', false)) {
```

Comment:
Make this a property that gets initialized by the config. Also, I'd put the check in the call above rather than here. Someone may want to just call this method.